portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,36 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  String [] comments = {
+      "Wow good job on the first portfolio website",
+      "You should find some help, the design is wacky",
+      "Where am I?"
+  };
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String json = convertToJson(comments);
+    
     response.setContentType("text/html;");
-    response.getWriter().println("<H1>Hello Raymond, this is a message!</H1>");
+    response.getWriter().println(json);
+  }
+
+  private String convertToJson(String[] comments) {
+    String json = "{";
+    json += "\"comment_" + 1 + "\": ";
+    json += "\"" + comments[0] + "\"";
+    json += ", ";
+    json += "\"comment_" + 2 + "\": ";
+    json += "\"" + comments[1] + "\"";
+    json += ", ";
+    json += "\"comment_" + 3 + "\": ";
+    json += "\"" + comments[2] + "\"";
+    json += "}";
+    return json; 
   }
```

Comment:
The Java Library provides classes you can import to build the JSON:
https://docs.oracle.com/javaee/7/api/javax/json/JsonArray.html

A quick Google search can help you find some examples.

Reply:
added maven dependency to project! Had fun learning 💁